app/scripts/modules/core/src/pipeline/manualExecution/ManualPipelineExecutionModal.tsx

```diff
@@ -364,7 +364,7 @@ export class ManualExecutionModal extends React.Component<IManualExecutionModalP
                   )}
                   {formik.values.pipeline && (
                     <div className="form-group">
-                      <div className="col-md-6 col-md-offset-4">
+                      <div className={pipelineOptions.length > 0 ? 'col-md-6 col-md-offset-4' : 'col-md-10'}>
```

Comment:
after changing away from `col-md-10` in the last PR, I decided that each view (manual execution w/ pipeline picker + manual execution of a specific pipeline) needed it's own layout to look right.